### ISSUE TITLE
the training loss value using your pre-trained model

### ISSUE THREAD

**Issue by congjianting** [pudae/tensorflow-pspnet#5]
hi, 

 @pudae , what is your last loss  value  when the iou value = 0.346 in the val datasets?
